scalableminds/webknossos
# get rid of deprecation and other warnings

### Note: needs updates in kube repo
- update application.conf in kube repo (play.crypto.secret + slick.db)

### URL of deployed dev instance (used for testing):
- https://deprecationwarnings.webknossos.xyz

### Steps to test:
- slick db connection should still work (you’ll notice quickly if it doesn’t)
- edit datasource error reporting should still work (e.g. readable error message if you edit the datasource json to have zero layers)

### Issues:
- fixes #2531 
- fixes #3058 

------
